decentraland/AppNavigator.js

```diff
@@ -16,34 +16,64 @@ import EthereumQuestionScreen from "./screens/EthereumQuestionScreen";
 import EthereumSignUpScreen from "./screens/EthereumSignUpScreen";
 import EthereumSignInScreen from "./screens/EthereumSignInScreen";
 import MyAssetsScreen from "./screens/MyAssetsScreen";
+
 import Colors from "@constants/Colors.js";
+import { responsiveHeight } from "react-native-responsive-dimensions";
+
+const defaultNavigationOptions = {
+  headerStyle: {
+    backgroundColor: Colors.headerBackgroundColor,
+    height: responsiveHeight(7),
+  },
+};
 
-const defaultNavigationOptions = ({ navigation }) => {
+const headerWithMenuButton = ({ navigation }) => {
   return {
     headerLeft: (
       <Button transparent onPress={() => navigation.toggleDrawer()}>
         <Icon name="menu" />
       </Button>
     ),
-    headerStyle: {
-      backgroundColor: Colors.headerBackgroundColor,
-    },
-    headerTintColor: Colors.headerTintColor,
-    headerTitleStyle: {
-      fontWeight: "bold",
-    },
+  };
+};
+
+const headerWithBackButton = ({ navigation }) => {
+  return {
+    headerLeft: (
+      <Button transparent onPress={() => navigation.goBack()}>
+        <Icon name="md-arrow-back" />
```

Comment:
Should we have it say the word `Back` too in addition to the icon?

Reply:
I've read this on my mobile Gmail app and it's the same their (without text) I think that is standard enough?

Reply:
I would argue that if we want a really mainstream audience we need to reach people who use the default email client on their device (I’d have to check how the back button looks there though).

Airbnb almost always supplements their icons with text in case people would get confused.

But either way we’d need designs for how this looks, and it’s far from the top priority, so I’ll keep it in mind and maybe create an issue for it

Reply:
Resolving this